### ISSUE TITLE
Use different font for specified language.

### ISSUE THREAD

**Issue by Kimi-Arthur** [etigerstudio/typora-misty-theme#14]
First, thanks a lot for the eye-pleasing theme!

There are cases where part of the note is in different language, thus the characters appear differently even though they have the same unicode. Normally this can be achieved by specifying a `lang` attribute.

It works for like Github theme (like below), but not for this beautiful theme.

Example:

<span lang="ja-JP">器</span> is different from 器

Source code:
```html
<span lang="ja-JP">器</span> is different from 器
```

**Comment by etigerstudio** [comment]
Thank you for pointing this subtle issue out. I will investigate🕵️‍♂️ on this right now.

**Comment by etigerstudio** [comment]
After some digging, I found that this issue was tightly related to the preset font fallback list, especially on Chinese fonts. 

## The font issue

As of now, Misty enforce "PingFang SC" on macOS as well as "Microsoft YaHei" on Windows. For this reason, exact Japanese kanji or even Korean hanja and traditional Chinese character may not be able to be rendered as expected because glyphs of those preset fonts take precedence.

To resolve this issue & recover the precedence, phasing out those preset fonts from fallback list is one feasible method, which is also preferred for I realized it's unnecessary to hard-code fonts if OS already got a correct default choice.

I confirmed macOS automatically choose "PingFang" for sans-serif font, so it's safe to remove that font from list. While I'm **not sure** whether it's safe to remove "Microsoft YaHei" on Windows without introducing breaking change such as falling back to "SimSun", which could be a major visual regression.

## Patched theme

I uploaded one patched theme. You could download and test out whether it works. Any screenshots will be helpful to me to understand original/patched rendering effect on **your** computer.

Original theme: [misty-original.txt](https://github.com/etigerstudio/typora-misty-theme/files/4303561/misty-original.txt)
Patched theme: [misty-patched.txt](https://github.com/etigerstudio/typora-misty-theme/files/4303562/misty-patched.txt)
Note: remember to change file extension to "css" from "txt".

## My screenshots

```html
<span lang="ja-JP">器</span> is different from 器
<span lang="ja-JP">今</span> is different from 今
<span lang="ja-JP">世界</span> is different from 世界
<span lang="ko">器</span> is different from 器
<span lang="ko">今</span> is different from 今
<span lang="ko">世界</span> is different from 世界
<span lang="zh-Hant">器</span> is different from 器
<span lang="zh-Hant">今</span> is different from 今
<span lang="zh-Hant">世界</span> is different from 世界
```

**Windows original:**
<img width="832" alt="Screen Shot 2020-03-09 at 12 26 41 AM" src="https://user-images.githubusercontent.com/2973222/76167841-b2c4e380-61a4-11ea-97aa-4abb360c468b.png">
**Windows patched** (noticing that Chinese font reverted to "SimSun", which might not happen on your computer):
<img width="820" alt="Screen Shot 2020-03-09 at 12 25 38 AM" src="https://user-images.githubusercontent.com/2973222/76167840-b0fb2000-61a4-11ea-8389-b5e3f55ffee7.png">

**macOS original:**
<img width="233" alt="Screen Shot 2020-03-09 at 1 21 48 AM" src="https://user-images.githubusercontent.com/2973222/76167842-b35d7a00-61a4-11ea-8fab-f2040119d2e5.png">
**macOS patched:**
<img width="211" alt="Screen Shot 2020-03-09 at 1 21 55 AM" src="https://user-images.githubusercontent.com/2973222/76167843-b3f61080-61a4-11ea-925b-4e622e47f0a6.png">

**Comment by Kimi-Arthur** [comment]
I get similar results. Here is my screenshot on Windows:
![无标题](https://user-images.githubusercontent.com/2446305/76253442-025dea80-624b-11ea-930d-32f7bd55e345.png)

It's the thing you get with Github too. So I guess it's the way to be?

**Comment by Kimi-Arthur** [comment]
Actually, installing the language pack helps a bit. Here is what I get after installing the Japanese language pack.
![无标题](https://user-images.githubusercontent.com/2446305/76254162-443b6080-624c-11ea-8a27-d5ae3ba3fe16.png)

**Comment by etigerstudio** [comment]
👍LGTM.
We might hold on for a little bit more time while I roll out a fresh Windows installation to test whether I could achieve similar results to yours. (Currently, my Windows 10 build is 1703 which is far behind the state-of-the-art environment.)

**Comment by etigerstudio** [comment]
Things have gone right for me on a fresh Win10 with language pack installed. I'm beginning to work on a new release that includes this fix.

Baseline (with GitHub):
<img width="220" alt="Screen Shot 2020-03-10 at 4 16 30 PM" src="https://user-images.githubusercontent.com/2973222/76301205-d8630180-62f8-11ea-8164-74259050975e.png">

Original Misty:
<img width="208" alt="Screen Shot 2020-03-10 at 4 16 22 PM" src="https://user-images.githubusercontent.com/2973222/76301217-dc8f1f00-62f8-11ea-95b3-fe6a036978a2.png">

Patched Misty:
<img width="210" alt="Screen Shot 2020-03-10 at 4 16 04 PM" src="https://user-images.githubusercontent.com/2973222/76301220-de58e280-62f8-11ea-84bc-37be6b92cd2e.png">

**Comment by etigerstudio** [comment]
Fixed in [`v1.0-alpha.2`](https://github.com/etigerstudio/typora-misty-theme/releases/tag/v1.0-alpha.2).
